test/runtests.jl

```diff
@@ -244,8 +244,8 @@ float_types = [Float32, Float64]
         end
 
         @testset "Incompressibility" begin
-            for Nt in [1, 10, 100]
-                @test incompressible_in_time(CPU(), Float64, Nt)
+            for ft in float_types, Nt in [1, 10, 100, 1000]
```

Comment:
1000 time steps of 32^3 is an expensive test. Does the error change with the number of time-steps? Why is 1 time-step not sufficient to assess the error?

Reply:
True. I scaled it down to 100 time steps. The error does still accumulate (see comment above) but even if it didn't, would be good to make sure that the error after 100 time steps is still low (or machine epsilon) as expected.

100 time steps at 32^3 takes ~0.45 seconds so shouldn't be too bad.

Reply:
Is that also true on a travis VM?

Reply:
If the error accumulates linearly or in some other predictable manner then the number of time-steps taken is not an independent parameter of the test.

Reply:
Probably not but since the error at `Nt=10` is very close to the error at `Nt=1`, but does grow a little bit for `Nt=100`, I chose to go up to 100 time steps.

Reply:
> If the error accumulates linearly or in some other predictable manner then the number of time-steps taken is not an independent parameter of the test.

True. We only need to test `Nt=100`, but I'm keeping `Nt=1,10` and logging the error so we can see how much the error accumulates on different architectures just by looking at the Travis/GitLab build logs.

Reply:
Why would the error be different on different architectures? Is the error not associated with floating point round off error?